ci/scripts/run-tests.sh

```diff
@@ -19,6 +19,6 @@ pushd "$PROJECT_ROOT"
 
   bundle exec rake install
   bundle exec rake spec
-  bundle exec rake features
 
+  rake features
```

Comment:
Nitpicking here but it would look better with consistent calls to rake

Reply:
Not possible if you want to test vendored bundle projects. A `bundle exec` will always the local gem env and using the `with_original_env` method won't work because of RVM's env hacking explained in #531

Reply:
Actually, let me try one thing...

Reply:
Got it.